### PR TITLE
controller: add networkfence skeleton

### PR DESCRIPTION
**Pull request by Yuggupta27**
Add networkfence controller skeleton

Signed-off-by: Yug Gupta <yuggupta27@gmail.com>

### CODE REVIEW

**Comment by Yuggupta27** [comment]
Depends on #10

**Comment by Yuggupta27** [comment]
@Mergifyio rebase

**Review by Rakshith-R — DISMISSED** [review]
crd and manifests for networkfence api needs to be generated

File: controllers/networkfence_controller.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
```
Comment (nixpanic) [inline controllers/networkfence_controller.go:2]: you may now also update the year

Comment (Yuggupta27) [reply controllers/networkfence_controller.go]: Updated now :+1:

**Review by nixpanic — APPROVED** [review]
@mergifyio rebase

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Review by Madhu-1 — APPROVED** [review]

**Comment by Madhu-1** [comment]
@nixpanic looks like the status are not matching in the mergify rules https://github.com/csi-addons/kubernetes-csi-addons/pull/40/checks?check_run_id=4712851719

**Comment by nixpanic** [comment]
@Mergifyio rebase

**Pull request merged**